demo/components/presentational/LargeText.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { StyleSheet, Text } from "react-native";
+import { responsiveHeight } from "react-native-responsive-dimensions";
+import Colors from "../../constants/Colors";
```

Comment:
I created an issue for myself to add a name in package.json for these https://github.com/tasitlabs/tasit/issues/27